library/psa_crypto_storage.c

```diff
@@ -249,6 +255,7 @@ static psa_status_t psa_crypto_storage_get_data_length(
 typedef struct {
     uint8_t magic[PSA_KEY_STORAGE_MAGIC_HEADER_LENGTH];
     uint8_t version[4];
+    uint8_t lifetime[sizeof( psa_key_lifetime_t )];
     uint8_t type[sizeof( psa_key_type_t )];
     uint8_t policy[sizeof( psa_key_policy_t )];
```

Comment:
Should we store uint8_t attributes[] instead of splitting it up into a separate type and policy?

Reply:
See https://github.com/ARMmbed/mbed-crypto/pull/157#discussion_r306865039